build.sbt

```diff
@@ -103,6 +103,7 @@ lazy val plugin = project
     javaOnlySettings,
     moduleName := "semanticdb-javac",
     javaToolchainVersion := "1.8",
+    javacOptions += "-g",
```

Comment:
Is this required for the fix or it it only needed for debugging?

Reply:
Not required, but makes intellij not show a constant warning in the variables view while debugging. 
Unless you've a reason we shouldn't include it, I dont see it as anything but benign

Reply:
Sounds good, was just curious if this flag affected semanticdb